filament/backend/include/backend/BufferDescriptor.h

```diff
@@ -27,6 +27,14 @@
 namespace filament {
 namespace backend {
 
+/**
+ * A CPU memory-buffer descriptor, typically used to transfer data from the CPU to the GPU.
+ *
+ * A BufferDescriptor owns the memory buffer it references, therefore BufferDescriptor cannot
```

Comment:
Minor nit, but "owns the memory" seems like strong language since you could pass null for the callback and use BufferDescriptor to wrap a global buffer (e.g. for full-screen triangle).  Users can also make it point to the inside of a large reference-counted buffer simply by writing a special callback (which we do in gltfio).  Same goes for `PixelBufferDescriptor`.